src/webapp-core.sln

```diff
@@ -33,9 +31,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{CBB0AA48
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WaCore.Crud.UnitTests", "WaCore.Crud.UnitTests\WaCore.Crud.UnitTests.csproj", "{13973730-A24F-4059-81C9-BFE300EC406D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WaCore.Web", "WaCore.Web\WaCore.Web.csproj", "{6BDA7B6E-2853-415E-9C27-EAE603AAB7BD}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WaCore.Web", "WaCore.Web\WaCore.Web.csproj", "{6BDA7B6E-2853-415E-9C27-EAE603AAB7BD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WaCore.Sample.Middlewares", "WaCore.Sample.Middlewares\WaCore.Sample.Middlewares.csproj", "{504C2CFA-A7D3-4D11-A1B9-0CB4D78521E0}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WaCore.Sample.Middlewares", "WaCore.Sample.Middlewares\WaCore.Sample.Middlewares.csproj", "{504C2CFA-A7D3-4D11-A1B9-0CB4D78521E0}"
```

Comment:
I know it's not a part of this pull request, but the word 'middleware' is uncountable, so it does not have a plural form. Just like software, hardware and firmware.

Reply:
Created an issue for it: #46